packages/rspeedy/plugin-react/src/entry.ts

```diff
@@ -84,30 +82,15 @@ export function applyEntry(
       // We would like to avoid adding `__background` to the output CSS filename.
       const mainThreadEntry = `${entryName}__main-thread`
 
-      const mainThreadName = path.posix.join(
-        isLynx
-          // TODO: config intermediate
-          ? DEFAULT_DIST_PATH_INTERMEDIATE
-          // For non-Lynx environment, the entry is not deleted.
-          // So we do not put it in the intermediate.
-          : '',
-        `${entryName}/main-thread.js`,
-      )
+      const mainThreadName = `${entryName}/main-thread.js`
```

Comment:
I would like to keep the `.rspeedy` for JS chunks (at lease for the default value).

We may remove that when `chunkSplit.strategy !== 'all-in-one'`